### ISSUE TITLE
Needs to access 10.0.0.1 when switching to different Access Points to synchronize sessions/cookies. 5.1.2-beta.16-081922

### ISSUE THREAD

**Issue by darkhoundz** [darellep/adopisoft-beta-releases#4]
Affected clients:

- Subscriptions without expiration/pause.
- Time Based clients without pause

Temporary fix:

- Enabled pause time to those sessions affected
- Enabled 'Auto-pause device session when inactive or disconnected from the WiFi'
- Auto-resume when device reconnects to WiFi

Side note: Ang Auto-Pause feature, minsan gumagana sa aking Poco F3, minsan naman hindi. 2/10 tries bago gumana.

![2022-08-28 23_55_32-Admin _ Dashboard](https://user-images.githubusercontent.com/28075740/187083301-b043e148-7f2e-40d9-a805-d14cdefcf354.png)

**Comment by darellep** [comment]
Hello Sir @darkhoundz , 

Pag -inaaccess nyo po yung 10.0.0.1, nagsysync naman yung sessions? Nakakapaginternet na yung device?
This scenario is expected po kasi especially with different APs, need talaga mag access ng captive portal para magsync.

**Comment by darkhoundz** [comment]
> Hello Sir @darkhoundz ,
> 
> Pag -inaaccess nyo po yung 10.0.0.1, nagsysync naman yung sessions? Nakakapaginternet na yung device? This scenario is expected po kasi especially with different APs, need talaga mag access ng captive portal para magsync.

Yes sir nag sy-sync naman po. Pero as of now, guds na after ko ma enable ang auto pause. Di na sila nag lologin pa sa 10.0.0.1 para bumalik ang internet kahit mag switch pa sila sa ibang APs. 

Yung mga naka subcription based naman na walang pause time, need pa nila mag access sa 10.0.0.1 after lumipat. 

Overall, malaking improvement ito compare sa v5.1.1 na need pa mag bukas ng portal every time na lumipat sa ibang AP.

Salamat sir!
